<launch>
        <arg name="left_cam_name" value="left_eye"/>
        <arg name="camera_name" value="endoscope"/>
        <arg name="robot_arm_name" value="PSM1"/>

    <node pkg="teleop_vision" type="robot_to_camera_aruco" name="robot_to_camera_aruco" output="screen">

        <param name="cam_name" value="$(arg left_cam_name)"/>
        <param name="cam_pose_topic_name" value="/$(arg camera_name)/left/pose"/>
        <param name="robot_pose_topic_name" value="/dvrk/$(arg robot_arm_name)/position_cartesian_current"/>
        <param name="image_transport_namespace" value="/$(arg camera_name)/left/image_raw"/>
        <param name="number_of_calibration_points" value="12"/>
        <param name="visual_axis_length" value="0.06"/>
        <param name="show_image" value="true"/>
    </node>

</launch>
